cmd/pFaTools/entropyTrack.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/dna/pDna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta/pFasta"
+	"github.com/vertgenlab/gonomics/wig"
+	"log"
+	"math"
+	"os"
+)
+
+// EntropyTrackSettings defines the usage settings for the pFa entropyTrack subcommand.
+type EntropyTrackSettings struct {
+	InFile       string
+	OutFile      string
+	DefaultValue float64 // default value for wig.
+}
+
+// entropyTrackUsage defines the usage statement for the pFaTools entropyTrack subcommand.
+func entropyTrackUsage(entropyTrackFlags *flag.FlagSet) {
+	fmt.Printf("pFaTools entropyTrack - a tool for plotting positional information on sequence\n" +
+		"uncertainty from an input pFa file.\n" +
+		"Usage:\n" +
+		"pFaTools entropyTrack in.pFa out.wig" +
```

Comment:
This will work for a pFasta file with an arbitrary number of sequences. Each sequence will generate a unique Wig object, but all will be written to the same output file. Wigs are also cattable, so this can be run on chromosomes in separate pFa files and then outputs can be merged.